### PR TITLE
Bug 1144500 - Ignore pointer events on SHB bar

### PR DESCRIPTION
**Pull request by mikehenrty**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dc4b6758199018f59916fb5c028efc214dd78127

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5ed02ee789d3a66d6f98ef08441adb4b9b5db41b

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b5e357e8573b8d90974dfa0d81263e75689c7947

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5c62e813848e952bb00885d0473aa70c2542c1eb

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=41c086f00cafe3dc0c4cf24f5b2c2685d6a13859

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bda52a965fff6dbf2e821619578d2e395e0362f4

File: apps/system/js/software_button_manager.js [hunk 1]
```diff
@@ -352,9 +352,12 @@
       }
     },
 
-    // We just add this handler to prevent fuzzing issues where tapping the SHB
-    // will trigger events in the current app.
-    _emptyEventHandler: function() {
+    /**
+     * Used to prevent taps on the SHB container from stealing focus.
+     * @memberof SoftwareButtonManager.prototype
```
Comment (etiennesegonzac) [inline apps/system/js/software_button_manager.js]: let's keep a mention of fuzzing in the comment since it (hopefully) still applies.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=405ebea24b97926f307a708af7739f62928e2395

**Pull request merged**